backend/api/tests.py

```
@@ -0,0 +1,43 @@
+
+    def test_register_user(self):
+        """ Test User Registration """
+        data = {
+            "name": "John Doe",
+            "email": "john@example.com",
+            "password": "securepassword123"
+        }
+        response = self.client.post(self.register_url, data)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def test_login_user(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_login_user`

Add return type annotation: `None`

(ANN201)